src/components/Sidebar/Sidebar.jsx

```diff
@@ -48,7 +49,7 @@ export default class Sidebar extends Component {
                   title={ page.title }
                   anchors={ page.anchors }
                   currentPage= { currentPage }
```

Comment:
Can you please clean extra whitespace here? ~`= {`~ => `={`

Reply:
guess it can be automated later on with prettier